browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -129,6 +129,17 @@ SkPath BraveVerticalTabStyle::GetPath(
         hit_test_outsets.set_top(0);
       }
 
+      // We also want the first tab (taking RTL into account) to be selectable
+      // in maximized or fullscreen mode by clicking at the very edge of the
+      // screen.
+      if (ShouldExtendHitTest() && tab()->controller()->IsTabFirst(tab())) {
+        if (base::i18n::IsRTL()) {
```

Comment:
I think using something like `tab()->GetMirrored()` would be better.

Reply:
Done - thanks!